custom/icds_reports/reports/disha.py

```diff
@@ -39,12 +41,21 @@ def export_exists(self):
         else:
             return False
 
-    def get_json_export(self):
+    def get_export_as_http_response(self, stream=False):
```

Comment:
`get_http_response` should just do.

Reply:
I hope keeping it explicit is not an issue.